modules/apps/29-fee/keeper/escrow.go

```diff
@@ -54,16 +56,16 @@ func (k Keeper) DistributeFee(ctx sdk.Context, refundAcc, forwardRelayer, revers
 	// get module accAddr
 	feeModuleAccAddr := k.authKeeper.GetModuleAddress(types.ModuleName)
 
-	// send receive fee to forward relayer
-	err := k.bankKeeper.SendCoins(ctx, feeModuleAccAddr, forwardRelayer, feeInEscrow.Fee.ReceiveFee)
+	// send receive fee to dest relayer
```

Comment:
I think we discussed this before but I've forgotten. Is this the terminology we're aligning on? Just wondering :) 

@AdityaSripal

Reply:
forward/reverse relayer is my preference. I find source/destination can become confusing

Reply:
yeah agreed